model/project.go

```diff
@@ -491,6 +492,7 @@ type TaskGroup struct {
 	// data about the task group
 	MaxHosts              int             `yaml:"max_hosts" bson:"max_hosts"`
 	SetupGroupFailTask    bool            `yaml:"setup_group_can_fail_task" bson:"setup_group_can_fail_task"`
+	TeardownGroupFailTask bool            `yaml:"teardown_group_can_fail_task" bson:"teardown_group_can_fail_task"`
```

Comment:
TeardownGroup is actually something else and won't work for this, so this is misleading. It's actually TeardownTask (i.e. post tasks) that we're allowing to fail (not the best naming scheme)